mod/wikirate_source/set/type_plus_right/source/file/import.rb

```diff
@@ -108,6 +115,10 @@ def default_import_args args
     nest card.left.metric_card, :view=>:edit_in_form
   end
 
+  view :hidden_field do |args|
+    hidden_field_tag :is_metric_import_update, 'true'
+  end
```

Comment:
this is a very generic name for such a specific view.  I get that it's a narrow set, but it could still be confusing.